src/components/Styles/Home.css

```diff
@@ -1,196 +1,196 @@
-*{
-    padding: 0;
-    margin: 0;
-    box-sizing: border-box;
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
 }
 
 .header {
-    width: 100%;
-    height: 300px;
-    background: url('https://media.giphy.com/media/m3XqQ8QhuIUuQau7n5/giphy.gif'),linear-gradient(90deg, rgb(0, 0, 0) 0%, rgba(48,9,117,1) 0%, rgba(50,9,121,0.8687850140056023) 0%, rgba(46,8,115,1) 31%, rgba(39,7,101,1) 37%, rgba(48,9,118,1) 43%, rgba(38,7,99,1) 43%, rgba(41,7,105,1) 46%, rgba(46,8,114,1) 50%, rgba(44,13,118,1) 54%, rgba(38,27,129,1) 59%, rgba(50,9,120,1) 65%, rgba(41,23,127,1) 67%, rgba(47,11,119,1) 70%, rgba(36,60,152,1) 77%, rgba(0,212,255,1) 99%);
-    background-repeat: no-repeat;
-    background-size: contain;
+  width: 100%;
+  height: 300px;
+  background: url('https://media.giphy.com/media/m3XqQ8QhuIUuQau7n5/giphy.gif'), linear-gradient(90deg, rgb(0, 0, 0) 0%, rgba(48, 9, 117, 1) 0%, rgba(50, 9, 121, 0.8687850140056023) 0%, rgba(46, 8, 115, 1) 31%, rgba(39, 7, 101, 1) 37%, rgba(48, 9, 118, 1) 43%, rgba(38, 7, 99, 1) 43%, rgba(41, 7, 105, 1) 46%, rgba(46, 8, 114, 1) 50%, rgba(44, 13, 118, 1) 54%, rgba(38, 27, 129, 1) 59%, rgba(50, 9, 120, 1) 65%, rgba(41, 23, 127, 1) 67%, rgba(47, 11, 119, 1) 70%, rgba(36, 60, 152, 1) 77%, rgba(0, 212, 255, 1) 99%);
+  background-repeat: no-repeat;
+  background-size: contain;
 }
 
 .logo-container {
-    display: flex;
-    width: 100%;
-    justify-content: space-between;
-    padding: 20px;
-    background-color: #335CFA;
-    height: 50px;
-    font-family: 'Lobster', cursive;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  padding: 20px;
+  background-color: #335cfa;
+  height: 50px;
+  font-family: 'Lobster', cursive;
 }
 
 .logo {
-    font-size: 25px;
-    cursor: pointer;
-    align-self: center;
+  font-size: 25px;
+  cursor: pointer;
+  align-self: center;
 }
 
 .logo span {
-    color:rgba(240, 248, 255, 0.932);
+  color: rgba(240, 248, 255, 0.932);
 }
 
 .user-icon {
-    width: 25px;
-    cursor: pointer;
-    border-radius: 50%;
-    align-self: center;
+  width: 25px;
+  cursor: pointer;
+  border-radius: 50%;
+  align-self: center;
 }
- 
+
 .header-text {
-    float: right;
-    color: #fff;
-    font-weight: bold;
-    font-size: 35px;
-    margin: 7%;
-    font-family: 'Open Sans', sans-serif;
-    text-align: center;
+  float: right;
+  color: #fff;
+  font-weight: bold;
+  font-size: 35px;
+  margin: 7%;
+  font-family: 'Open Sans', sans-serif;
+  text-align: center;
 }
 
 .samples-container {
-    display: grid;
-    grid-template-columns: repeat(3, 1fr);
-    grid-template-rows: 1fr;
-    width: auto;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: 1fr;
+  width: auto;
 }
 
 .search-container {
-    display: flex;
-    background-color: #000;
-    justify-content: space-between;
-    padding: 1%; 
-    color: aliceblue;
-    font-size: 25px;
+  display: flex;
```

Comment:
- [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
This is just an example if you have same CSS rules please combine them.
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)